src/MapIt/index.js

```diff
@@ -291,7 +298,20 @@ function MapIt({
   ]);
   const classes = useStyles();
 
-  return <div id={mapId} className={classes.root} />;
+  return (
+    <Fragment>
+      {!featuresToDraw && (
+        <div className={classes.root}>
+          <BlockLoader loading width="100%" height="100%" />
+        </div>
+      )}
+      <div
```

Comment:
This couldn't be a `<BlockLoader ...><div ... /></BlockLoader>`

Reply:
I didn't do this because of the root div. 
Also because initial loading will look for div with id.
I will track this am make enhancement to the code later.